content/en/docs/installation/compatibility.md

```diff
@@ -40,9 +40,13 @@ To address this, the webhook can be run in the host network so it can be reached
 by cert-manager, by setting the `webhook.hostNetwork` key to true on your
 deployment, or, if using Helm, configuring it in your `values.yaml` file.
 
-Note that since kubelet uses port `10250` by default on the host network, the
-`webhook.securePort` value must be changed to a different, free port.
+**IMPORTANT**: kubelet on AWS uses port `10250` by default on the host network, which
```

Comment:
Suggestion/nit: 'AWS' -> 'AWS EKS'

Reply:
Arrgh need more coffee 😆 
Ignore the above nit- kubelet always uses port 10250 (by default) https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
I suggest to remove the 'on AWS' bit 😄